.github/workflows/workflow_pr.yml

```
@@ -272,12 +280,13 @@ jobs:
     needs:
     - create_env
     - comment_ref
-    uses: m10-payments/workflow-templates/.github/workflows/reusable_integration_tests.yml@v1.25.3
+    uses: m10-payments/workflow-templates/.github/workflows/reusable_integration_tests.yml@feat/INFRA-268_auth-in-admin
```

Сначала PR с изменениями, потом проставления тегов, потом PR с обновлением версий
```suggestion
    uses: m10-payments/workflow-templates/.github/workflows/reusable_integration_tests.yml@v1.25.3
```

---

Вообще я обхожу этот процесс иногда: навешиваю тег на своей фиче ветке, ставлю правильные версии в PR, а после мерджа перевешиваю тег с фиче ветки на мастер. Но в таком случае есть даунтайм